BranchSDK-Samples/Windows/ColorPicker/ColorPicker.cpp

```diff
@@ -0,0 +1,370 @@
+// ColorPicker.cpp : Defines the entry point for the application.
+//
+
+#include "framework.h"
+#include "ColorPicker.h"
+
+#include "commctrl.h"
+#include "commdlg.h"
+
+#include <BranchIO/Branch.h>
+#include <BranchIO/Event/StandardEvent.h>
+
+#define MAX_LOADSTRING 100
+
+// Global Variables:
+HINSTANCE hInst;                                // current instance
+HWND hwndMain;									// The Main Window
```

Comment:
formatting of "// The Main Window"

Reply:
Tabs vs. Spaces.  Reformatted entire document.